src/pages/Lend/Filter/SelectedRefinements.vue

```diff
@@ -1,38 +1,45 @@
 <template>
 	<div class="selected-refinements">
-		<ais-current-refinements
-			:transform-items="transformItems"
-		>
+		<ais-current-refinements :transform-items="transformItems">
 			<div slot-scope="{ items }">
-				<div class="filter-summary-container">
-					<div>{{ items.length }} Filters Applied</div>
-					<div v-if="isCollapsible" class="show-toggle-container">
-						<div v-if="isCollapsed" @click="handleClickShowMore">Show</div>
-						<div v-else @click="handleClickShowFewer">Hide</div>
+				<div class="row align-middle">
```

Comment:
The "Show all # filters" link should always alight to the top rather than the middle, also based on the designs Medium up should be 1 row of filters with this button to the right. Both of these items make "alight-middle" unnecessary.